src/board.hpp

```diff
@@ -3,13 +3,15 @@
 
 #include "coords.hpp"
 #include "enums.hpp"
+#include <array>
+#include <vector>
 
 namespace TicTacToe {
     class Board {
     public:
         std::optional <Symbol> fieldContent(Coords coords);
 
-        std::vector<Coords[3]> winningSymbolLines();
+        std::vector<std::array<Coords, BOARD_WIDTH>> winningSymbolLines();
```

Comment:
@olcia303 lmk if these array changes are up to date with what you've got pending for commit

Reply:
Kinda, i added using BoardLine = std::array<Coords, BOARD_WIDTH> instead of putting it all in one line

Reply:
that's even better

Reply:
resolved - 8b202d6